Tools/README

```diff
@@ -15,11 +25,16 @@ i18n            Tools for internationalization. pygettext.py
                 and msgfmt.py generates a binary message catalog
                 from a catalog in text format.
 
+importbench     A set of micro-benchmarks for various import scenarios.
+
 iobench         Benchmark for the new Python I/O system. (*)
 
 msi             Support for packaging Python as an MSI package on Windows.
 
-parser          Un-parsing tool to generate code from an AST.
+nuget           A package manager for Microsoft . NET applications.
```

Comment:
Typo:

```suggestion
nuget           A package manager for Microsoft .NET applications.
```

But is this actually the NuGet package manager, or something else?

Reply:
Yes, i was thinking that this folder contains packages and scripts related to the NuGet package manager for .NET projects ?

Reply:
I retired the blank space like you suggested to me in the last commit, no changes here ?

Reply:
Okay, so maybe something like this?

"Files for the NuGet package manager for .NET."

Reply:
It's looks good, thank you !